docs/manual/config/index.rst

```diff
@@ -197,18 +198,24 @@ creating a `qtile.desktop
 
 A second way to start Qtile is a custom X session. This way allows you to
 invoke Qtile with custom arguments, and also allows you to do any setup you
-want (e.g. special keyboard bindings like mapping caps lock to control, setting
+want (e.g. special keyboardy to start Qtile is a custom X session. This way
+allows you to i bindings like mapping caps lock to control, setting
```

Comment:
I don't get this diff

Reply:
sorry, this is accidantly paste, but I understand it is in master now, so, will fix with next diff